gcs/CHANGES.md

```diff
@@ -1,5 +1,9 @@
 ### 2.1.7 - 2022-02-03
```

Comment:
Should this be 2.1.8? Also, should we update the date?

Reply:
Thanks for pointing this out Arun